lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journey-definitions.yaml

```diff
@@ -21,3 +33,13 @@ ADDRESS_AND_FRAUD:
           exitEventToEmit: end
         next:
           exitEventToEmit: next
+        not-found:
+          targetState: ERROR
+        temporarily-unavailable:
+          targetState: ERROR
+        fail-with-no-ci:
+          targetState: ERROR
+        enhanced-verification:
+          targetState: ERROR
+        access-denied:
+          targetState: ERROR
```

Comment:
Part of me is wondering if rather than having all this repetition on every state, we could define one parent state that has all events we know about with a target state of error. And then all other states inherit it and override just the events we expect to receive. Might make the expected behaviour for each state more obvious while still handling events that *could* be thrown.

Reply:
Yeah - that could be a reasonable compromise. Having the repetition like this (and the test to validate) forces you to think about every state for every CRI, but perhaps it's unnecessarily tedious!

(It's still a lot better than having ad-hoc events all over the place)

Reply:
This is probably where I'd be keen for @blakeyp 's opinion, since HILUX might be at the receiving end of most changes like this :)

Reply:
I've moved additional events into the parent: CRI_STATE, to complete the set